api/v1alpha1/hardwareclassification_types.go

```diff
@@ -234,6 +298,26 @@ type HardwareClassificationStatus struct {
 	ErrorType ErrorType `json:"errorType,omitempty"`
 	// ProfileMatchStatus identifies whether a applied profile is matches or not
 	ProfileMatchStatus ProfileMatchStatus `json:"profileMatchStatus,omitempty"`
+	// The count of matched Hosts per profile reported by hardwareclassification system
+	MatchedCount MatchedCount `json:"matchedCount,omitempty"`
+	// The count of unmatched Hosts per profile reported by hardwareclassification system
+	UnmatchedCount UnmatchedCount `json:"unmatchedCount,omitempty"`
+	// The count of Hosts in error state
+	ErrorHosts ErrorHosts `json:"errorHosts,omitempty"`
+	// The count of hosts in registration error state
+	RegistrationErrorHosts RegistrationErrorHosts `json:"registrationErrorHosts,omitempty"`
+	// The count of hosts in introspection error state
+	IntrospectionErrorHosts IntrospectionErrorHosts `json:"introspectionErrorHosts,omitempty"`
+	// The count of hosts in provisioning error state
+	ProvisioningErrorHosts ProvisioningErrorHosts `json:"provisioningErrorHosts,omitempty"`
+	// The count of hosts in power management error state
+	PowerMgmtErrorHosts PowerMgmtErrorHosts `json:"powerMgmtErrorHosts,omitempty"`
+	// The count of hosts in Provisioned Registration error state
+	ProvisionedRegistrationErrorHosts ProvisionedRegistrationErrorHosts `json:"provisionedRegistrationErrorHosts,omitempty"`
```

Comment:
ditto!

Reply:
@digambar15, yes we need it because BMH/metal3 have introduced multiple more error host fields. Now HWCC is supporting all erro host type provided by BMH. Please have a look https://github.com/metal3-io/baremetal-operator/blob/master/apis/metal3.io/v1alpha1/baremetalhost_types.go +137